.rodare.json

```diff
@@ -1,15 +1,68 @@
 {
-    "creators": [
+    "creators":[
         {
-            "name": "Widera, René",
-            "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
-            "orcid": "0000-0003-1642-0459"
+            "name":"Widera, René",
+            "affiliation":"Helmholtz-Zentrum Dresden-Rossendorf",
+            "orcid":"0000-0003-1642-0459"
+        },
+        {
+            "name":"Bastrakov, Sergei",
+            "affiliation":"Helmholtz-Zentrum Dresden-Rossendorf",
+            "orcid":"0000-0003-3396-6154"
+        },
+        {
+            "name":"Kelling, Jeffrey",
+            "affiliation":"Helmholtz-Zentrum Dresden-Rossendorf",
+            "orcid":"0000-0003-1761-2591"
+        },
+        {
+            "name":"Ehrig, Simeon",
+            "affiliation":"Helmholtz-Zentrum Dresden-Rossendorf",
+            "orcid":"0000-0002-8218-3116"
+        },
```

Comment:
This is not alphabetical

Reply:
In another universe it is^^ Checked only the contributors and forgot the creators^^

Reply:
fixed